discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -196,7 +198,7 @@ def set_olm_operator(self, operator_name, properties=None):
             olm_operators.append({"name": operator.name, "properties": operator.properties})
         olm_operators.append({"name": operator_name, "properties": properties})
 
-        self.update_config(olm_operators=olm_operators)
+        self.nodes.controller.config.olm_operators = olm_operators
```

Comment:
i don't understand why it is why operators are part of node.controller.config

Reply:
You right